src/main/resources/META-INF/plexus/components.xml

```diff
@@ -17,6 +22,7 @@
        <package>org.apache.maven.plugins:maven-jar-plugin:2.6:jar,${project.groupId}:${project.artifactId}:${project.version}:build</package>
        <pre-integration-test>${project.groupId}:${project.artifactId}:${project.version}:start</pre-integration-test>
        <integration-test>org.apache.maven.plugins:maven-failsafe-plugin:2.19.1:integration-test</integration-test>
+       <!-- post-integration-test is opportunity for code coverage, such as jacoco, to collect statistics from running container -->
```

Comment:
Ok, got your intention to leave the hook free for others to kick in. However, putting it into `verify` has the nasty side effect that `mvn integration-test` wont call this phase so the container keeps running. It also *feels* more natural to put it into `pre` and `post` goals. Couldn't a user still bind to the `post-integration-test` phase so that both goals are called ?